src/System.Threading.AccessControl/src/Configurations.props

```diff
@@ -1,14 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project DefaultTargets="Build">
   <PropertyGroup>
     <PackageConfigurations>
-      net461-Windows_NT;
       netstandard-Windows_NT;
       netstandard;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations)
-      netfx-Windows_NT;
```

Comment:
Same here. Maybe add a placeholder configuration for netfx so that we test against the inbox implementation.

Reply:
Also, we use the `PackageConfigurations` and `BuildConfigurations` properties all together, whenever we want to do a separation in between what we build for the package (shipping assets) and when we want to build a configuration in the vertical for testing. Whenever we don't have this separation, like now in this case, we just use `BuildConfigurations`. Whenever there is no `PackageConfigurations` property defined, we use `BuildConfigurations` as the `PackageConfigurations` for that package.